packages/frontend/src/components/DashboardTiles/EmptyStateNoTiles/index.tsx

```diff
@@ -32,23 +33,25 @@ const RunQueryButton: FC<{ projectId: string }> = ({ projectId }) => (
     </ButtonWrapper>
 );
 
-const NoSavedChartsAvailable = () => (
+const NoSavedChartsAvailable = ({ isChart = false }) => (
```

Comment:
We can make this two components, one specifically for charts and one for dashboards. There won't be much duplicate code.